test/parse.js

```diff
@@ -0,0 +1,433 @@
+				},
+				{
+					args: "color(acescg 0 1 .5)",
+					expect: '{"spaceId":"acescg","coords":[0,1,0.5],"alpha":1}'
+				},
+				{
+					args: "color(acescg 0 100% 50%)",
+					expect: '{"spaceId":"acescg","coords":[0,1,0.5],"alpha":1}'
+				},
+				{
+					args: "color(xyz 0 1 .5)",
+					expect: '{"spaceId":"xyz-d65","coords":[0,1,0.5],"alpha":1}'
+				},
+				{
+					args: "color(xyz 0 100% 50%)",
+					expect: '{"spaceId":"xyz-d65","coords":[0,1,0.5],"alpha":1}'
+				},
+				{
+					args: "color(xyz-d65 0 1 .5)",
+					expect: '{"spaceId":"xyz-d65","coords":[0,1,0.5],"alpha":1}'
+				},
+				{
+					args: "color(xyz-d65 0 100% 50%)",
+					expect: '{"spaceId":"xyz-d65","coords":[0,1,0.5],"alpha":1}'
+				},
+				{
+					args: "color(xyz-d50 0 1 .5)",
+					expect: '{"spaceId":"xyz-d50","coords":[0,1,0.5],"alpha":1}'
+				},
+				{
+					args: "color(xyz-d50 0 100% 50%)",
+					expect: '{"spaceId":"xyz-d50","coords":[0,1,0.5],"alpha":1}'
+				},
+				{
+					args: "color(jzazbz 0 25% -50%)",
+					expect: '{"spaceId":"jzazbz","coords":[0,0.125,-0.25],"alpha":1}'
+				},
+				{
+					args: "color(--jzazbz 0 25% -50%)",
+					expect: '{"spaceId":"jzazbz","coords":[0,0.125,-0.25],"alpha":1}'
+				},
+				{
+					args: "color(--hct 0.25turn 50% 25)",
+					expect: '{"spaceId":"hct","coords":[90,72.5,25],"alpha":1}'
+				},
+				{
+					name: "With transparency",
+					args: "color(display-p3 0 1 0 / .5)",
+					expect: '{"spaceId":"p3","coords":[0,1,0],"alpha":0.5}'
+				},
+				{
+					name: "No arguments",
+					args: "color(display-p3)",
+					expect: '{"spaceId":"p3","coords":[0,0,0],"alpha":1}'
+				},
+				{
+					name: "No arguments / alpha",
+					args: "color(display-p3 / .5)",
+					expect: '{"spaceId":"p3","coords":[0,0,0],"alpha":0.5}'
+				},
+				{
+					name: "Fewer arguments",
+					args: "color(display-p3 1)",
+					expect: '{"spaceId":"p3","coords":[1,0,0],"alpha":1}'
+				},
+				{
+					name: "Fewer arguments / alpha",
+					args: "color(display-p3 1 / .5)",
+					expect: '{"spaceId":"p3","coords":[1,0,0],"alpha":0.5}'
+				},
+				{
+					name: "none red",
+					args: "color(display-p3 none 1 .5)",
+					expect: '{"spaceId":"p3","coords":[null,1,0.5],"alpha":1}'
+				}
+			]
+		},
+		{
+			name: "hsl()",
+			tests: [
+				{
+					name: "hsl(), commas",
+					args: "hsl(180, 50%, 50%)",
+					expect: '{"spaceId":"hsl","coords":[180,50,50],"alpha":1}'
+				},
+				{
+					name: "hsl(), negative hue",
+					args: "hsl(-180, 50%, 50%)",
+					expect: '{"spaceId":"hsl","coords":[-180,50,50],"alpha":1}'
+				},
+				{
+					name: "hsl(), hue > 360",
+					args: "hsl(900, 50%, 50%)",
+					expect: '{"spaceId":"hsl","coords":[900,50,50],"alpha":1}'
+				},
+				{
+					name: "hsla(), degrees for hue, spaces and slash",
+					args: "hsl(90deg 0% 0% / .5)",
+					expect: '{"spaceId":"hsl","coords":[90,0,0],"alpha":0.5}'
+				},
+				{
+					name: "hsla(), rad for hue, spaces and slash",
```

Comment:
This (and some of the ones after) mention `hsla()` but  use `hsl()`, is that what the original tests did?

Reply:
That's what the original tests did.